usaspending_api/common/helpers/sql_helpers.py

```diff
@@ -17,6 +18,14 @@
 TYPES_TO_QUOTE_IN_SQL = (str, datetime.date)
 
 
+def get_database_dsn_string(force_default=False):
```

Comment:
What is the impetus for the force_default logic? It's not used in this PR.

Reply:
It can be useful when testing locally by want to run scripts with IS_LOCAL=False